student/src/main/java/tds/student/services/remote/RemoteResponseService.java

```diff
@@ -49,7 +62,21 @@ public PageGroup insertItems(final OpportunityInstance oppInstance, final Adapti
 
   @Override
   public PageList getOpportunityItems(final OpportunityInstance oppInstance, final boolean validate) throws ReturnStatusException {
-    return legacyResponseService.getOpportunityItems(oppInstance, validate);
+    PageList pageList = null;
+
+    if (isLegacyCallsEnabled) {
+      pageList = legacyResponseService.getOpportunityItems(oppInstance, validate);
+    }
+
+    if (!isRemoteExamCallsEnabled) {
+      return pageList;
+    }
+
+    List<ExamPage> examPagesWithItems = examPageRepository.findAllPagesWithItems(oppInstance);
+
+    List<OpportunityItem> opportunityItems = getOpportunityItemsFromExamItems(examPagesWithItems);
```

Comment:
Two things, can we rename this method to convertPagesToPageList() and instead just have it return a PageList.  I don't see a reason to break apart the conversion on line 77 and 79.  Unless you see another instance in this class were we convert items and don't want a PageList.